eng/common/TestResources/New-TestResources.ps1

```diff
@@ -62,7 +62,10 @@ param (
     [switch] $CI = ($null -ne $env:SYSTEM_TEAMPROJECTID),
 
     [Parameter()]
-    [switch] $Force
+    [switch] $Force,
+
+    [Parameter()]
+    [switch] $File
```

Comment:
Name is too vague. Maybe `OutFile`, which is what a lot of built-in cmdlets use. As written, not clear if it's input or output. `-OutFile` is typical when you intend to write out to a file.

Reply:
Changed to -OutFile